lib/src/components/card_rounded.dart

```diff
@@ -35,11 +50,17 @@ class CardRounded extends StatelessWidget {
     this.height,
     this.width,
     this.onClose,
+    this.closeButtonAlignment = Alignment.topRight,
+    this.customButtonIcon = SFSymbols.xmark,
+    this.padding = const EdgeInsets.symmetric(
+      vertical: 12,
+      horizontal: 12,
+    ),
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    final Widget _closeButton;
+    final Widget _positionedButton;
```

Comment:
This is still a "close" button, is it not?

```suggestion
    final Widget _closeButton;
```